pyproject.toml

```diff
@@ -17,7 +17,18 @@ test-command = "pytest {project}/tests"
 [tool.cibuildwheel.macos]
 archs = ["x86_64", "universal2", "arm64"]
 
-# Reduce the size of Linux wheels.
-[[tool.cibuildwheel.overrides]]
-select = "*-manylinux_*"
-environment = { CXXFLAGS="-g0" }
+# Try caching compilation of harfbuzz.
+[tool.cibuildwheel.linux]
+before-build = [
+    "yum install -y ccache",
+    "echo '#!/bin/sh\nccache gcc $@' > /usr/local/bin/gcc-ccache",
+    "chmod +x /usr/local/bin/gcc-ccache",
+    "echo '#!/bin/sh\nccache g++ $@' > /usr/local/bin/g++-ccache",
+    "chmod +x /usr/local/bin/g++-ccache",
```

Comment:
In Debian, there is a dir you add to the PATH after installing ccache so that cc, gcc etc are all symlinks to ccache. Fedora/RH probably have something similar.

Reply:
Ah, I see. The directory seems to be empty though after installation on the manylinux 2014 image?!

Reply:
I don’t know, but if this works then 🤷🏽‍♂️

Reply:
I'm trying to do better but it seems to be slower now, got 60s per wheel build at some point and now it's 90s. Not sure if this is the config or GHA being slower for some reason...